src/services/car/moneybackApplication.ts

```diff
@@ -1,30 +1,43 @@
-import { postData, handleValidationError, Service } from "../../base"
+import { postData, handleValidationError, ApiCallOptions } from "../../base"
 
 import { WithValidationError } from "../../types/withValidationError"
 import { MoneybackApplication } from "../../types/models/applications"
 
-export const sendMoneybackApplication = async (
-  listingId: number,
-  moneybackApplication: MoneybackApplication,
-  options = {}
-): Promise<WithValidationError<MoneybackApplication>> => {
-  const { validateOnly, recaptchaToken, validateFields } = {
-    validateOnly: false,
-    recaptchaToken: null,
-    validateFields: Object.keys(moneybackApplication),
-    ...options,
+export const sendMoneybackApplication = async ({
+  listingId,
+  moneybackApplication,
+  options = {},
+}: {
+  listingId: number
+  moneybackApplication: MoneybackApplication
+  options: ApiCallOptions & {
```

Comment:
I would avoid combining request options with sepecific options for the concrete endpint

Reply:
Why? So that consumer needs to know that `recaptchaToken` is passed in the second option object and `validateOnly` in the first one? That creates unnecessary complexity and makes usage cumbersome in my opinion.

Reply:
I don't think it matters a lot at the end  so up to you...

My thinking behind would be to get it somewhat close to the request or fetch interface where the first argument is the resource and the second one the options (I realized body is an option there...). It's probably preference but thats like how I am used to interfaces of this kind... (It's kinda separation of concerns as the consumer could always pass the same request options interface no matter of the specific api request; I do see some benefits when it comes to maintanance as the generic request interface could live standalone. Given the very unlikely case that we want to add `validateOnly` that does somthing different than the option in this request for example )

Reply:
Seems like premature optimization to me... Let's cross the clashing options bridge when we get there ok?
I see your point but it does require the consumer to be aware of what is a custom option and what is a specific option and this means that consumer needs to know the implementation details which sounds a bit like smell to me...